server/src/test/java/org/apache/iotdb/db/integration/IoTDBCreateTimeseriesIT.java

```diff
@@ -65,25 +62,30 @@ public void tearDown() throws Exception {
   }
 
   /** Test creating a time series that is a prefix path of an existing time series */
+  @Ignore // nested measurement has been forbidden
   @Test
   public void testCreateTimeseries1() throws Exception {
     String[] timeSeriesArray = {"root.sg1.aa.bb", "root.sg1.aa.bb.cc", "root.sg1.aa"};
 
-    for (String timeSeries : timeSeriesArray) {
-      statement.execute(
-          String.format(
-              "create timeseries %s with datatype=INT64, encoding=PLAIN, compression=SNAPPY",
-              timeSeries));
-    }
+    try {
+      for (String timeSeries : timeSeriesArray) {
+        statement.execute(
+            String.format(
+                "create timeseries %s with datatype=INT64, encoding=PLAIN, compression=SNAPPY",
+                timeSeries));
+      }
 
-    // ensure that current timeseries in cache is right.
-    createTimeSeries1Tool(timeSeriesArray);
+      // ensure that current timeseries in cache is right.
+      createTimeSeries1Tool(timeSeriesArray);
```

Comment:
？ space

Reply:
Maybe, the code is aligned with the context. I think this doesn't matter and could pass spotless.